src/_blog/libp2p-and-Ethereum.md

```diff
@@ -0,0 +1,311 @@
+
+### GossipSub v1.1
+Updates to GossipSub and its formalization efforts also made significant headway.
+Protocol Labs:
+- Introduced [GossipSub v1.1](https://github.com/libp2p/specs/pull/254) which hardended the protocol to prevent a wide-range of attacks against it (see this [blog post](https://blog.ipfs.tech/2020-05-20-gossipsub-v1.1/) to learn more).
+- Published an [evaluation report of GossipSub v1.1](https://research.protocol.ai/publications/gossipsub-v1.1-evaluation-report/vyzovitis2020.pdf).
+- Tested the protocol with a new framework called **[Testground](https://ipfs.io/ipfs/bafybeigyfbsv5rbudh6ufyahfjb4c4fbgayg7cmrrbhnmppni3tdtsg66q/2020-05-06-launching-testground/)** (related article: [Highlights of Testground 2022](https://blog.ipfs.tech/testground-highlights-in-2022/)).
+
+Similarly, the Ethereum 2.0 team:
+- Proposed [improvements to the peer scoring functionality](https://github.com/ethereum/consensus-specs/issues/1761).
+- Updated the Beacon Chain specifications to [formally require GossipSub v1.1](https://github.com/ethereum/consensus-specs/pull/1796) (after the peer scoring improvements were added)
+- Added to the GossipSub [specs](https://github.com/libp2p/specs/pull/294) and [code](https://github.com/libp2p/go-libp2p-pubsub/pull/359) to prevent network splits and avoid privacy problems in Eth 2.0
+
+<div class="container" style="display:flex; column-gap:10px;">
+    <figure>
+        <img src="https://i.imgur.com/tYwvEIA.png">
+    </figure>
+</div>
+
+<!-- ![](https://i.imgur.com/tYwvEIA.png) -->
+
+Finally, in December 1, 2020, the Beacon Chain genesis occurred!:
+<blockquote class="twitter-tweet"><p lang="en" dir="ltr">Congratulations on genesis, <a href="https://twitter.com/hashtag/eth2?src=hash&amp;ref_src=twsrc%5Etfw">#eth2</a>, from your gossipy and streamy cousin libp2p! 🚀<br><br>Eth2 has been a driving force in the <a href="https://twitter.com/hashtag/libp2p?src=hash&amp;ref_src=twsrc%5Etfw">#libp2p</a> community:<br><br>* catalysing py-libp2p, nim-libp2p and jvm-libp2p<br>* (co-)funding initiatives<br>* reporting vulnerabilities<br>* contributing R&amp;D<br><br>❤️🙌 <a href="https://twitter.com/ethereum?ref_src=twsrc%5Etfw">@ethereum</a></p>&mdash; libp2p (@libp2p) <a href="https://twitter.com/libp2p/status/1333761655881297921?ref_src=twsrc%5Etfw">December 1, 2020</a></blockquote> <script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
+
+### Beacon Chain Networking Spec 📝
+
+Since it's inception, the networking specification to the Beacon Chain saw a steady stream of changes and updates. After mid 2021 (August), the spec had mostly matured and the libp2p requirements were all locked in (in fact, there were no further updates until January 2022.)
```

Comment:
```suggestion
Since its inception, the networking specification of the Beacon Chain has seen a steady stream of changes and updates. After mid-2021 (August), the spec mainly had matured, and the libp2p requirements were all locked in (in fact, there were no further updates until January 2022.)
```